utilities/getRelativeDateString.ts

```diff
@@ -0,0 +1,50 @@
+export default function getRelativeDateString(date: Date): string {
```

Comment:
```suggestion
export function getRelativeDateString(date: Date): string {
```

Reply:
Or you can make `getRelativeDateString` and `getRelativeShortDateString` in separated files